Update authentication-context-example.md (#4023)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/authentication-context-example.md

Once the label has been updated, guests accessing a SharePoint site (or the **Files** tab in a team) with that label will be required to agree to the terms of use before gaining access to that site.

> ## Blocking background apps (rolloing out in preview)
> 
> If authentication context is set on a site then admins can choose to prevent background apps from accessing that site for the apps that is assigned with that authentication context in a conditional access policy. You can configure an conditinal access policy such that a specific authentication context can be assigned to choosen application principles (non Microsoft applications). You have to explicitly turn this feature on via below cmdlet. Note: You must have at least one conditional access policy with application principle configured.   

Reviewer: ```suggestion
## Blocking background apps (rolling out in preview)

If authentication context is set on a site, admins can choose to prevent background apps from accessing that site for the apps assigned with that authentication context in a conditional access policy. You can configure a conditional access policy such that a specific authentication context can be assigned to chosen application principles (non-Microsoft applications). You'll need to explicitly turn this feature on via the following cmdlet. Note that you should have at least one conditional access policy with an application principle configured.   
```
